### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by MatthiasWanner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,97 @@
...
 
+- run `$ git clone git@github.com:WildCodeSchool/btz-0321-aeviso-api.git` in your terminal
+- run `$ cd btz-0321-aeviso-api`
+- run `npm install `
+
+We use a POSTRGESQL database. If you already have it, paste the url in DATABASE_URL environment variable (in your .env file at the root of the project).  
+If not, a `docker-compose.yml` file can be used:
+
+- Be sure your docker app is running
+- In your terminal, run `docker-compose up -b`
```
Comment (Aeryle) [inline README.md:23]: The flag used is `-b` but it should be `-d`

File: README.md [hunk 2]
```diff
@@ -1,47 +1,97 @@
...
+ -ThomasBarrial: https://github.com/ThomasBarrial  
+ -MatthiasWanner: https://github.com/MatthiasWanner  
+ -Benwade-progldc: https://github.com/Benwade-prog
+
+API REST made for AEVISO client project: https://github.com/WildCodeSchool/btz-0321-aeviso-client.  
+With jwt authentication.
 
 ## Setup
 
+- run `$ git clone git@github.com:WildCodeSchool/btz-0321-aeviso-api.git` in your terminal
```
Comment (Aeryle) [inline README.md:15]: Maybe put an HTTPS version, or link a tutorial on how to setup a SSH key?

File: README.md [hunk 3]
```diff
@@ -1,47 +1,97 @@
...
+ -Aeryle: https://github.com/Aeryle  
+ -Chabelle78: https://github.com/Chabelle78  
+ -ThomasBarrial: https://github.com/ThomasBarrial  
+ -MatthiasWanner: https://github.com/MatthiasWanner  
+ -Benwade-progldc: https://github.com/Benwade-prog
+
+API REST made for AEVISO client project: https://github.com/WildCodeSchool/btz-0321-aeviso-client.  
+With jwt authentication.
 
 ## Setup
```
Comment (Aeryle) [inline README.md:13]: Maybe also add a "Prerequisites" category with different links to downloads (Node, Docker, etc) ?

File: README.md [hunk 4]
```diff
@@ -1,47 +1,97 @@
...
+http://localhost:5000
+
 ```
 
-## Test
+## Documentation
+
+go to `/api-docs`
+
+## Facultativs Variables
```
Comment (Aeryle) [inline README.md:77]: Small typo. Faculativs -> Facultatives. Also wouldn't "Optional" be a better name?

File: README.md [hunk 5]
```diff
@@ -1,47 +1,97 @@
...
+
+If you want to share API to your local network
+
+`HOST=ip.local.adress`
```
Comment (Aeryle) [inline README.md:81]: Small typo. Adress -> Address.

File: README.md [hunk 6]
```diff
@@ -1,47 +1,97 @@
...
+
+Check if the container is running now with an `adminer` image at `8080` port, and a `postgres` image at `5342` port.  
+Check if the `aeviso` database was created running `http://localhost:8080` in your browser.  
+If all is ok, you should be on the adminer login page. Complete the form like that:
+
+![Adminer](images/adminer_form.png)
+
+password: `aeviso`.
+
+`aeviso` database should appear in the list, but she's empty.
```
Comment (Chabelle78) [inline README.md:33]: "but it is empty"

**Review by Chabelle78 — APPROVED** [review]

**Review by Chabelle78 — APPROVED** [review]

**Pull request merged**